netdeploy/remote/deployedNetwork.go

```diff
@@ -283,9 +338,424 @@ func (cfg DeployedNetwork) BuildNetworkFromTemplate(buildCfg BuildConfig, rootDi
+
+	fmt.Printf("starts %v\n", time.Now())
+	accounts := make(map[basics.Address]basics.AccountData)
+
+	genesis, err := bookkeeping.LoadGenesisFromFile(genesisFolder + "/genesis.json")
+	if err != nil {
+		return err
+	}
+
+	var bootstrappedNet netState
+
+	bootstrappedNet.genesisID = genesis.ID()
+	bootstrappedNet.genesisHash = crypto.Hash([]byte(genesis.ID()))
+	srcWallet := getGenesisAlloc(fileCfgs.SourceWalletName, genesis.Allocation)
+	if srcWallet.Address == "" {
+		return fmt.Errorf("error finding source wallet address")
+
+	}
+
+	rewardsPool := getGenesisAlloc("RewardsPool", genesis.Allocation)
+	if rewardsPool.Address == "" {
+		return fmt.Errorf("error finding source rewards ppol address")
+
+	}
+
+	feeSink := getGenesisAlloc("FeeSink", genesis.Allocation)
+	if feeSink.Address == "" {
+		return fmt.Errorf("error finding fee sink address")
+
+	}
+	src, err := basics.UnmarshalChecksumAddress(srcWallet.Address)
+
+	poolAddr, err := basics.UnmarshalChecksumAddress(rewardsPool.Address)
+	if err != nil {
+		return err
+	}
+	sinkAddr, err := basics.UnmarshalChecksumAddress(feeSink.Address)
+	if err != nil {
+		return err
+	}
+
+	//initial state
+	bootstrappedNet.nAssets = fileCfgs.GeneratedAssetsCount
+	bootstrappedNet.nApplications = fileCfgs.GeneratedApplicationCount
+	//bootstrappedNet.nApplications = 0
+	bootstrappedNet.txnState = protocol.PaymentTx
+	bootstrappedNet.roundTxnCnt = fileCfgs.RoundTransactionsCount
+	bootstrappedNet.round = basics.Round(0)
+
+	params := config.Consensus[protocol.ConsensusCurrentVersion]
+	minAccounts := accountsNeeded(fileCfgs.GeneratedApplicationCount, fileCfgs.GeneratedAssetsCount, params)
+	nAccounts := fileCfgs.GeneratedAccountsCount
+	if minAccounts > nAccounts {
+		bootstrappedNet.nAccounts = minAccounts
+	} else {
+		bootstrappedNet.nAccounts = nAccounts
+	}
+
+	accounts[poolAddr] = basics.MakeAccountData(basics.NotParticipating, rewardsPool.State.MicroAlgos)
+	accounts[sinkAddr] = basics.MakeAccountData(basics.NotParticipating, feeSink.State.MicroAlgos)
+	//fund src account with enough funding
+	bootstrappedNet.fundPerAccount = basics.MicroAlgos{Raw: uint64(bootstrappedNet.nAssets) * params.MinBalance * 2}
+	totalFunds := srcWallet.State.MicroAlgos.Raw + bootstrappedNet.fundPerAccount.Raw*bootstrappedNet.nAccounts + bootstrappedNet.roundTxnCnt*fileCfgs.NumRounds
+	accounts[src] = basics.MakeAccountData(basics.Online, basics.MicroAlgos{Raw: totalFunds})
+
+	bootstrappedNet.poolAddr = poolAddr
+	bootstrappedNet.sinkAddr = sinkAddr
+
+	//gensis block
+	initState, err := generateInitState(accounts, &bootstrappedNet)
+	if err != nil {
+		return err
+	}
+	localCfg := config.GetDefaultLocal()
+	localCfg.Archival = true
+	localCfg.CatchpointTracking = -1
+	localCfg.LedgerSynchronousMode = 0
+	log := logging.NewLogger()
+	l, err := ledger.OpenLedger(log, genesisFolder+"/bootstrapped", false, initState, localCfg)
+	if err != nil {
+		return err
+	}
+	l.WaitForCommit(0)
```

Comment:
I'm surprised you found that to be needed. But it wouldn't be incorrect.

Reply:
it's not needed. I meant to remove it.